ddl/ddl_api.go

```diff
@@ -1561,7 +1562,8 @@ func (d *ddl) RecoverTable(ctx sessionctx.Context, tbInfo *model.TableInfo, sche
 		SchemaName: schema.Name.L,
 		Type:       model.ActionRecoverTable,
 		BinlogInfo: &model.HistoryInfo{},
-		Args:       []interface{}{tbInfo, autoID, dropJobID, snapshotTS, recoverTableCheckFlagNone},
+		Args: []interface{}{tbInfo, recoverInfo.CurAutoIncID, recoverInfo.CurAutoRandID,
```

Comment:
Have you check the compatibility? I guess there will have this problem. Maybe you should put `recoverInfo.CurAutoRandID` in the end to avoid this problem.

Reply:
Okay, let me put this to the end.